api/oneclick.py

```diff
@@ -670,12 +716,12 @@ def validate_item(self, licensepool):
             raise InvalidInputException("Licensepool %r doesn't know its ISBN.", licensepool)
 
         if licensepool.work and licensepool.work.presentation_edition:
-            media = licensepool.work.presentation_edition.media
+            media = licensepool.work.presentation_edition.medium
 
         return item_oneclick_id, media
 
 
-    def validate_patron(self, patron, create=False):
+    def validate_patron(self, patron, create=True):
```

Comment:
validate_patron was never being called with create=True. It was impossible to ever borrow a book because you'd never get a OneClick account. I don't see any reason to _ever_ call this without create=True, since we always want any interested patron to have a OneClick account, so I fixed it by changing the default.